docs/components/Table.js

```diff
@@ -26,23 +28,25 @@ data () {
     headers: [
       { field: 'id', title: 'ID', sortable: true },
       { field: 'first_name', title: 'First Name', sortable: true, hidden: true },
-      { field: 'last_name', title: 'Last Name', sortable: true, alignRight: true }
+      { field: 'last_name', title: 'Last Name', sortable: true, alignRight: true },
+      { field: 'friends', title: 'Friends', sortable: true }
     ],
     users: [
-      { 'id': 1, 'first_name': 'Jesse', 'last_name': 'Simmons' },
-      { 'id': 2, 'first_name': 'John', 'last_name': 'Jacobs' },
-      { 'id': 3, 'first_name': 'Tina', 'last_name': 'Gilbert' },
-      { 'id': 4, 'first_name': 'Clarence', 'last_name': 'Flores' },
-      { 'id': 5, 'first_name': 'Anne', 'last_name': 'Lee' },
-      { 'id': 6, 'first_name': 'Rick', 'last_name': 'Potter' },
-      { 'id': 7, 'first_name': 'Harry', 'last_name': 'Sanchez' },
-      { 'id': 8, 'first_name': 'Joy', 'last_name': 'Mann' },
-      { 'id': 9, 'first_name': 'Adam', 'last_name': 'Hansen' },
-      { 'id': 10, 'first_name': 'Marge', 'last_name': 'Grey' }
+      { 'id': 1, 'first_name': 'Jesse', 'last_name': 'Simmons', 'friends': '44' },
```

Comment:
for some reason, the sorting doesn't work if the numbers are strings. Making them number types like the id field is makes it sort on click of the header in the docs

Reply:
@Boshes seems like a bug in `Table.vue` . please report an issue and we can tackle it separately.

Reply:
It doesn't look like a bug in blaze but rather in how the implementation was done since the docs use lodash orderBy to sort the data. Looks like orderBy for lodash will sort by character codes when strings are provided for the ordering where it should be the number data type so it would be handled properly

Reply:
@Boshes I agree with you. Can this be considered outside scope of this pr. Please create an issue.

Reply:
Sure, it can be looked at with other documentation fixes in an issue after this gets put in, if everyone is cool with it as well